docs/BF Labeling Guide.md

```diff
@@ -0,0 +1,167 @@
+# Bot Framework Labeling Guidelines (DRAFT)
+
+This article describes the common labels used in the Bot Framework repositories to tag issues and PRs.
+
+It also provides guidance on how to use them.
+
+## Label format
+
+In order to be able to parse the different labels in reports we use `:` to separate the categories and subcategories in a label. The general format of a label is:
+
+```bash
+Main category:Subcategory 1:Subcategory 2
+```
+
+There's no limit to the number of subcategories but normally 2 or 3 levels at most should be enough.
+
+## Label usage
+
+Some labels can be applied multiple times to the same issue and some other shouldn't. For example, an issue could be labeled as `Area:Adaptive` **and** `Area:Skills` but it should **only** have one priority label `P:0` or `P:1`.
+
+## Label colors
+
+Labels should use the same colors across repos to improve readability and in most cases, all labels within the same main category should use the same color.
+
+## Main label categories
+
+This section describes the main label categories being used:
+
+|Category| Description  | Usage  | Example
+|---|---|---|------|
+|[Area](#area)|Defines a functional area or feature of the product.| Multiple |`Area:Skills`|
```

Comment:
The doc team went with two categories here, one for services and one for "components", since we handle each differently in the docs. I don't know if that distinction makes sense in the code repos.

Reply:
If possible I'd rather not add a new category, would creating subcategories under Area work?